OpenMEEGMaths/CMakeLists.txt

```diff
@@ -1,8 +1,57 @@
 # OpenMEEGMath
-add_subdirectory(include)
-add_subdirectory(src)
+add_library(OpenMEEGMaths SHARED
```

Comment:
here we should not add all the cpp files since not all of them should be public.